server/user_account/views.py

```diff
@@ -145,18 +142,29 @@ def remove_token(self, request): # pylint: disable=no-self-use
                         status=status.HTTP_200_OK)
 
 
-class AccessAllClients(generics.GenericAPIView):
-    queryset = CustomUser.objects.values(
-        'user_name',
-        'first_name',
-        'last_name',
-        'role',
-        'is_active',
-        'email',
-        )
-    http_method_names = ['get']
-
+class AccessAllClients(viewsets.ModelViewSet):
+    @api_view(['Get'])
+    @permission_classes([IsAuthenticated])
```

Comment:
Statements at line 146 and 147 should not be used in class based views, it is only used in function based views so it doesn't make sense to have it here. If you are a bit confused at the difference between class based views and function based views the tutorials on Django that @NicholasNagy and I did explains it really well (Or again you have the documentation.